test/unit/remote_execution_feature_test.rb

```diff
@@ -43,4 +43,28 @@
       composer.targeting.search_query.must_equal "name = #{host.name}"
     end
   end
+
+  describe '.register' do
+    it "creates a feature if it's missing" do
+      feature = RemoteExecutionFeature.register('new_feature_that_does_not_exist', 'name')
+      feature.must_be :persisted?
+      feature.label.must_equal 'new_feature_that_does_not_exist'
+      feature.name.must_equal 'name'
+      refute feature.host_action_button
+    end
+
+    it "creates a feature with host action flag" do
+      feature = RemoteExecutionFeature.register('new_feature_that_does_not_exist_button', 'name', :host_action_button => true)
+      feature.must_be :persisted?
+      assert feature.host_action_button
```

Comment:
I added separate test just to test the scope, I hope it makes you at least [this happy](https://www.youtube.com/watch?v=y6Sxv-sUYtM) :-)